SwanCustomEnvironments/swancustomenvironments/serverextension.py

```diff
@@ -0,0 +1,88 @@
+from tornado import web
+from traitlets import Unicode
+from traitlets.config import Configurable
+
+from jupyter_server.base.handlers import JupyterHandler, APIHandler
+from jupyter_server.utils import url_path_join
+
+import subprocess
+import re
+from os import path
+
+
+class SwanCustomEnvironments(Configurable):
```

Comment:
Please add some class docstring, even if short, to explain the purpose of the class.